.readthedocs.yaml

```diff
@@ -13,11 +13,6 @@ build:
 sphinx:
   configuration: docs/source/conf.py
 
-formats:
```

Comment:
This formats are mandatory to generate those documents.
It seems you didn't rebase with up-to-date dev branch.